src/controllers/api/inventoryController.ts

```diff
@@ -1,11 +1,28 @@
-import inventory from "@/static/fixed_responses/inventory.json";
+/* eslint-disable @typescript-eslint/no-misused-promises */
```

Comment:
isnt disabled globally now?

Reply:
no idea it shouldn't be

Reply:
then remove the one from .eslintrc

Reply:
I agree with you that it is stupid to have the line in every file that has an async handler.

But I don't like to globally disable a warning.  Ill turn the .eslintrc back on